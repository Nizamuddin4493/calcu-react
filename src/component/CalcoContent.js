import React from 'react';

const CalcoContent = () => (
  <>
    <div>
      <h1 className="ldsc">Lets do some Math...</h1>
    </div>
  </>
);

export default CalcoContent;
