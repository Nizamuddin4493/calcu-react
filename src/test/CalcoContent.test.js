import { BrowserRouter as Router } from 'react-router-dom';
import TestRenderer from 'react-test-renderer';
import CalcoContent from '../component/CalcoContent';

it('renders correctly', () => {
  const tree = TestRenderer.create(<Router><CalcoContent /></Router>).toJSON();
  expect(tree).toMatchSnapshot();
});
