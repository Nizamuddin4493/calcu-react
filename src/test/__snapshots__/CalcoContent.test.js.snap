// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div>
  <h1
    className="ldsc"
  >
    Lets do some Math...
  </h1>
</div>
`;
